.github/workflows/develop-cd.yml

```
@@ -0,0 +1,164 @@
+    needs: deploy
+    runs-on: ubuntu-latest
+    if: always()
+
+    steps:
+       - name: 배포 결과 디스코드 알림 전송
+         env:
+           DISCORD_WEBHOOK: ${{ secrets.DISCORD_WEBHOOK }}
+           STATUS: ${{ needs.deploy.result }}
+           COMMIT_MESSAGE: ${{ github.event.head_commit.message }}
+           AUTHOR: ${{ github.actor }}
+           IMAGE_TAG: ${{ needs.build-and-push.outputs.image-tag }}
```

actionlint (1.7.10)

property "build-and-push" is not defined in object type {deploy: {outputs: {}; result: string}}

(expression)